sqlite-android/src/main/java/io/requery/android/database/sqlite/SQLiteStatementType.java

```diff
@@ -65,34 +65,42 @@ public static int getSqlStatementType(String sql) {
             return STATEMENT_OTHER;
         }
         String prefixSql = sql.substring(0, 3).toUpperCase(Locale.US);
```

Comment:
I infer that you intended to do away with the toUpperCase for performance reasons (as you switched to comparing via equalsIgnoreCase on purpose per your comment), but you are still using toUpperCase here? Is that intended?

Reply:
You're write :(
Thanks